packages/design/src/tokens/dark.tokens.json

```diff
@@ -11,7 +11,7 @@
                     "$value": "hsl(197, 18%, 25%)"
                 },
                 "900": {
-                    "$value": "hsl(197, 15%, 18%)"
+                    "$value": "hsl(197, 15%, 16%)"
```

Comment:
Tamps down the `blue--900` lightness a pinch in dark mode to maintain an equivalent contrast ratio with the rest of the steps